cpp/assetmanager.cpp

```diff
@@ -118,4 +109,14 @@ void AssetManager::check_updates() {
 #endif
 }
 
+AssetManager::tex_ptr AssetManager::get_missing_tex() {
+	// Lazily load the missing texture.
+	if (!missing_tex) {
+		missing_tex.reset(new Texture{root->join("missing.png"), false});
```

Comment:
Changed for the sake of consistancy with the old code inside WITH_INOTIFY. Pointless imho.